### PR TITLE
basic readme

### PR DESCRIPTION
**Pull request by ghassanmas**
Relates #11

### CODE REVIEW

**Review by hasansaad1 — APPROVED** [review]

**Review by prodionov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+## How is the data visualized?:
+
+The main library that we use for data-visualization is [D3.js](https://d3js.org), also some of the charts are based on plain vanilla JS.
+
+For user interactions: **filtering** the data and filling the forms is done using the library [ReactJS](https://reactjs.org).  
+
+## Have a look
+
+
+[![The prototype](rikaz.png)](https://www.figma.com/proto/IeBFPegDJoYORSqJFwHXRKgc/Untitled?scaling=contain&node-id=4%3A2)
```
Comment (matthewdking) [inline README.md]: You shouldn't have pictures only used in your README in your file structure as they aren't necessary. Post the picture in an issue on Github and then get the link to that picture.

**Review by matthewdking — CHANGES_REQUESTED** [review]
It would be really good if we could see some user stories on the readme as well :+1:

File: README.md (hunk 1 above)
Comment (ghassanmas) [reply README.md]: Moved, certainly we will add more to the readme, I rushed since we don't wont to have empty readme :smile:

**Review by matthewdking — CHANGES_REQUESTED** [review]
Same comment as last time :point_up: 

Please include some user stories on your README

**Review by matthewdking — APPROVED** [review]
LGTM :taco:

**Pull request merged**